assets/js/main.js

```diff
@@ -49,23 +50,26 @@ import(/* webpackChunkName: "vue-apps" */ './vue-apps/index').then(vueApps => {
     vueApps.init();
 });
 
-const isDoNotTrack =
-    window.doNotTrack === '1' ||
-    window.navigator.doNotTrack === '1' ||
-    window.navigator.msDoNotTrack === '1';
+function shouldInitAnalytics() {
+    const isDoNotTrack =
+        window.doNotTrack === '1' ||
+        window.navigator.doNotTrack === '1' ||
+        window.navigator.msDoNotTrack === '1';
 
-/* Disable analytics outside of the real domain to avoid polluting data. */
-function shouldBlockAnalytics() {
-    const isProduction = window.AppConfig.environment === 'production';
-    const isProdDomain =
-        window.location.hostname === 'www.tnlcommunityfund.org.uk';
-    return !isProduction && !isProdDomain;
+    if (window.AppConfig.environment === 'production') {
```

Comment:
The logic here was getting a little bit "will you heed my rhyming verse, and take my answer in reverse: when is the hostname not-not the one for which I search" so I've tried to make it more direct.

Reply:
Great improvement 👍